### DISCUSSION TITLE
Siemens data

### DISCUSSION THREAD

**Discussion by rosenph** [QSMxT/QSMxT#63] [General]
Having succeeded with Philips' data, I tried a single-echo Siemens SWI sequence similar to the test data provided. Although the test data worked, my data did not. Here is the output data from the NeuroDesk. Can you please help?
[Error Siemens Output3.odt](https://github.com/QSMxT/QSMxT/files/9357839/Error.Siemens.Output3.odt)

**Comment by stebo85** [comment]
Dear @rosenph,

It looks like there is no data in your dicom folder, because it only finds 2 files. Can you share the dicom data, then we could have a look?

Cheers
Steffen

**Reply by rosenph** [reply]
Dear Steffen,
The DICOM data are two directories down from the DICOM file. Here are the dcm files. I had to delete 16 of them to reduce the size.
Regards,
Philip.
[DICOM.zip](https://github.com/QSMxT/QSMxT/files/9358289/DICOM.zip)

**Reply by stebo85** [reply]
Dear @rosenph,

There are 2 issues with your data:

1) your dicoms lack a file ending. QSMxT looks for various file endings to identify dicom data, but no file-ending is a problem. You can fix this by running:
```
 for file in `ls`; do mv $file ${file}.IMA; done
```
in your dicom folder or by exporting your dicom data from the scanner with a file ending (IMA, ima, DCM, dcm ...)

2) the second issue is that you have filtered phase data where crucial information was already destroyed by the SWI preprocessing - so you cannot use this data for QSM processing.

You can identify filtered phase data by the absence of phase wraps:
<img width="705" alt="image" src="https://user-images.githubusercontent.com/4021595/185094478-fde832b2-8dd6-434e-8544-bdb3be8d72b3.png">

This is how phase data should look like:
<img width="177" alt="image" src="https://user-images.githubusercontent.com/4021595/185094707-adccaec3-f672-4c99-a36c-3fcb60072c03.png">

Check if you have the unprocessed phase data as dicoms on the scanner. If not, you could retrospectively reconstruct raw .dat files on the Siemens system to give you raw phase.


Thank you
Steffen

**Reply by rosenph** [reply]
Dear Steffen,
Thank you very much for your help. I had the same problem with the Philips files, but I missed the unwrapping  in the Siemens' menu.

I found that I could not add a file ending with " do mv $file ${file}.IMA " but I found, after a search , that "  rename -- 's/$/.IMA' *  " did, and in batch.

Thank you again for your help.

Regards,

Philip.

**Reply by stebo85** [reply]
Dear @rosenph,

Great to hear that this is working for you now. If you can't get unfiltered phase from the swi protocol you are using, you can also just run a plain gre sequence and make sure to use adaptive combine on Siemens instead of sum-of-squares coil combination.

Thank you
Steffen
